laravel/app/Models/Assignment.php

```diff
@@ -19,4 +19,29 @@ class Assignment extends Model
      * @var bool
      */
     public $timestamps = false;
+
+    public function residents()
+    {
+        return $this->belongsTo(Resident::class);
+    }
+
+    public function options()
```

Comment:
should be singular `option`, since an assignment only belongs to one Option. I'm not 100%, but I think it's a 1 to 1 relationship?